app/profile/[id]/createTask/page.tsx

```diff
@@ -20,8 +20,8 @@ export default function CreateTask() {
   const [task, setTask] = useState<TaskType>({
     title: "",
     description: "",
-    category: CategoryTasksType.OTHER,
-    importanceLevel: ImportanceLevelTasksType.NO_IMPORTANT,
+    category: SelectCategoryTaskType.DEFAULT,
+    importanceLevel: SelectImportanceLevelTasksType.DEFAULT,

```

Comment:
Jeżeli zapiszesz ten obiekt jako zmienną, nawet ponad komponentem, to możesz ją reużyć tutaj oraz w poniżśzym przypadku gdzie ustawiasz chyba default value (linia 64-68)